scripts/npm/get-owners.js

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ Simple script to return npm maintainers of Backpack in
+ a form easily digestible by a bash script.
+ */
+
+const meta = require('../../meta.json');
+
+meta.maintainers.forEach(maintainer => {
+  process.stdout.write(`${maintainer.npm}\n`);
+});
```

Comment:
The output of this script is a replacement for the `NPM_OWNERS` file. I suppose it's a bit of a hack but it was significantly easier than trying to parse JSON in Bash, or rewriting all of our Bash scripts into JS.